journal-parsing/src/main/java/io/debezium/ibmi/db2/journal/retrieve/RetrieveJournal.java

```diff
@@ -111,9 +111,9 @@ public boolean retrieveJournal(JournalProcessedPosition previousPosition, final
         builder.init();
         builder.withBufferLenth(config.journalBufferSize());
         builder.withJournalEntryType(JournalEntryType.ALL);
-        if (config.filtering() && !config.includeFiles().isEmpty()) {
-            builder.withFileFilters(config.includeFiles());
-        }
+        // if (config.filtering() && !config.includeFiles().isEmpty()) {
```

Comment:
Please remove the commented out code if needed.
But I guess it should not be removed as such but change elswher is probbaly needed.

Reply:
You might consult @msillence and explain why you need it and you migh find a different solution.